app/events/views.py

```diff
@@ -161,7 +146,35 @@ def my_rsvps():
 		return jsonify({"message" : "you have not responded to any events yet"}), 200
 	return jsonify({"message" : "please login or signup to see your rsvps"}), 401
 
-
-
-
-
+@events.route('/search', methods=['POST'])
```

Comment:
would it be better if you could separate the routes to categories and location individually, in a scenario where the location is the same as the event it will only show the location results.

Reply:
i will refactor that and see what is best doing